kn/fotos/api.py

```diff
@@ -61,12 +62,18 @@ def _list(data, request):
     user = request.user if request.user.is_authenticated() else None
     return album_json(album, user)
 
-def _set_title(data, request):
+def _set_metadata(data, request):
     if 'title' not in data:
         return {'error': 'missing title attribute'}
     if not isinstance(data['title'], basestring):
         return {'error': 'title should be string'}
-    title = data['title']
+    title = data['title'] or None
```

Comment:
Hoe zou jij het dan doen? Helemaal uitschrijven? 
```python
if not title.strip():
    title = None
```
Ik zie liever geen lege strings in de database als daar ook `null` in staat.